sim/src/config.h

```diff
@@ -73,13 +73,15 @@ namespace crimson {
       uint server_count;
       uint server_iops;
       uint server_threads;
+      bool server_use_heap;
```

Comment:
No. This variable has no usage as of now, but will come handy (during testing, debugging) if we want dmclock to switch between two data-structure (heap, linear) dynamically.